nexus-sdk/src/main/java/io/nexusrpc/handler/OperationStartDetails.java

```diff
@@ -87,11 +101,20 @@ public Builder setRequestId(String requestId) {
       return this;
     }
 
+    /** Add a link. */
+    public Builder addLinks(Link link) {
```

Comment:
Also add a `getLinks` here so they can mutate in more advanced ways if they'd like

(same for OperationStartResult)

Reply:
done in https://github.com/nexus-rpc/sdk-java/pull/12/commits/7dad38525e4bf8d031033f4de17d8dfb9dae5d8b